include/APIClient/Router.hpp

```diff
@@ -5,24 +5,27 @@
 
 using PutFormData = std::map<std::string, std::string>;
 
+namespace APIClient {
 class Router {
```

Comment:
I'm not sure why you are using a class instantiable for that. The `Router` doesn't hold any state in the instance, it's probably better to use a non-instantiable class with just `static` methods. (or better, a `namespace`)